Podcast/UserSettings.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  UserSettings.swift
+//  Podcast
+//
+//  Created by Drew Dunne on 10/23/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class UserSettings: NSObject {
+    
+    static let sharedSettings = UserSettings()
+    
+    private var saveCounter: Int!
+    private let maxSave: Int = 5
+    
+    var pushNotifications: Bool!
+    
+    private override init() {
+        saveCounter = 0
+        super.init()
+        loadData()
+    }
+    
+    private func loadData() {
+        pushNotifications = UserDefaults.standard.bool(forKey: "push_notifications")
+    }
+    
+    func writeToFile() {
+        UserDefaults.standard.set(pushNotifications, forKey: "push_notifications")
+    }
+    
+    static let changeUsernamePage: SettingsPageViewController = {
```

Comment:
let's comment these two static fields a bit because its getting a bit confusing how all these components are interacting together

Reply:
And okay will do. I'm working on a better way to define these.